docs/experiments-plus/monitor.md

```diff
@@ -18,7 +18,8 @@ To monitor the status of your experiment,
     - **Checks have valid unit type** ensures that config checks contain the unit ID type selected for this experiment (user ID by default).  Available when checks begin.
     - **Event metrics have data** checks that events are being logged with unit IDs matching the experiment.  If logged events don't have the same unit ID as the experiment exposures, event based metrics won't be available in Pulse. A common cause is running an experiment using an ID type that isn't userID (eg. stableID or a customID) - while events data coming from a system like Segment is missing this identifier to match to. You can look at examples of how to set this [here](/integrations/data-connectors/segment#user-ids-and-custom-ids). Available within an hour after an exposure and event have been logged by the same unitID.
     - **Pulse metrics available** monitors availability of Pulse results, which are expected on the day after the experiment starts.  
-    - **Exposures are balanced** checks that the number of units exposed in each group matches the expected allocation.  The Sample Ratio Mismatch check is performed using a Chi-Squared test of independence. A p-value < 0.01 triggers a warning for possible imbalance that should be monitored, check again the following day. P-value < 0.001 means there is likley a problem with the experiment exposures.       
+    - **Exposures are balanced** checks that the number of units exposed in each group matches the expected allocation.  The Sample Ratio Mismatch check is performed using a Chi-Squared test of independence. A p-value < 0.01 triggers a warning for possible imbalance that should be monitored, check again the following day. P-value < 0.001 means there is likley a problem with the experiment exposures.      
+    - **Parameter changed** warns you when a parameter was changed on an experiment that was in progress.  This could mean that users in one or more groups are receiving a different experience than they were at the beginning of the experiment, making it a different experiment.  In those cases, we recommend restarting the experiment to get an accurate measure of results.  However, than can be some valid reason to update paramters - for example, if you are adding a certain parameter that controls an experience that was supposed to trigger at a certain time, and adding it later was the same as creating it at the beginning since it would not have changed the experience until that point. Hence it is only flagged as a warning. If the change did not impact the experiment, you can ignore the warning. 
```

Comment:
- "receiving a difference experience" -> "receiving different experiences" ? since users plural. maybe, I don't know how grammar works
- paramters -> parameters
- I would drop the explanation for valid cases, and just say it's up to them to decide whether or not they are okay with it, which is why it's a warning and not an error